TheMET/TheMET/Extentions/UISearchBar+apply.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UISearchBar+apply.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 03/08/2023.
+//
+
+import Foundation
+import UIKit
+
+extension UISearchBar {
+    
+    func apply(barTintColor: UIColor?,  textFieldBackgroundColor: UIColor?, textFieldColor: UIColor?) {
+        if let barTintColor = barTintColor,
+           let textFieldBackgroundColor = textFieldBackgroundColor,
+           let textFieldColor = textFieldColor,
+           let textField = self.value(forKey: "searchField") as? UITextField {
```

Comment:
`value(forKey:)`/`setValue(forKey:)`  is unsafe legacy way to access properties in **ObjC**.

They must not be used in modern code if unnecessary. Please use `self.searchTextField` instead.